File: spark-tutorials/src/main/java/com/getting_start/java/SparkContextTest.java
package com.getting_start.java;
import org.apache.spark.SparkConf;
import org.apache.spark.api.java.JavaSparkContext;


public class SparkContextTest {
	public static void main(String args[]){
		SparkConf conf = new SparkConf().setMaster("local").setAppName("My App");
		JavaSparkContext sc = new JavaSparkContext(conf);
		sc.stop();
	}
}

File: spark-tutorials/src/main/scala/com/getting_start/scala/SparkContextTest.scala
import org.apache.spark.SparkContext
import org.apache.spark.SparkContext._
import org.apache.spark.SparkConf

object SparkContextTest {
  def main(args: Array[String]){
    val conf = new SparkConf().setMaster("local").setAppName("My App")
    val sc = new SparkContext(conf)
    sc.stop()
  }
}